### DISCUSSION TITLE
Error: ENOENT: no such file or directory, open ...\cypress.json

### DISCUSSION THREAD

**Discussion by aleberman** [cypress-io/cypress#15061] [Cypress Cloud]
I'm trying to run a .js file script in cypress but when I run it it keeps throwing the same error: "Error writing to: C:\Users\...\cypress\integration\examples\prueba1.js\cypress.json

Error: ENOENT: no such file or directory, open 'C:\Users\...\cypress\integration\examples\prueba1.js\cypress.json'"

My test file (prueba1.js) is in the same integration folder as the example files. Though I do can run the example files provided during the installation of cypress, and I can see the code in the Visio editor, as soon as I save the .js file, it does not appear in cypress

Help pls! Tks!!

**Comment by bahmutov** [comment]
I am not sure what is going wrong with your machine, Cypress expects the following files when running in folder "my-folder" for example

```
my-folder/
   cypress.json
   cypress/
      fixtures/
      integration/
         your spec files
      plugins
      support
```

You should be in the "my-folder" folder when calling `npx cypress open` or `npx cypress run` command
